tox.ini

```diff
@@ -29,6 +29,18 @@ extras =
 commands =
     pytest {posargs}
 
+[testenv:coverage]
```

Comment:
does it make sense to run the tests a second time? should we simply run `coverage`?

Reply:
I think running coverage is enough because the VM used for this is almost the same one that comes default with cairo (the difference, for now, is just storing the touched program counters), so tests are run for coverage in the same environment in practice.